cla_backend/apps/call_centre/sql/ExportProviderStaff.sql

```diff
@@ -0,0 +1,13 @@
+COPY (SELECT
+id,
+created,
+modified,
+user_id,
+provider_id,
+is_manager,
+chs_organisation,
+chs_user,
```

Comment:
not actually sure they need `chs_organisation` and `chs_user`, maybe we can delete them